api/fixtures/example.go

```diff
@@ -11,6 +11,11 @@ import (
 	crclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
```

Comment:
Platforms [already have constant types](https://github.com/openshift/hypershift/blob/745a6c3a4b6f098281e56623fbfdc7f89784e171/api/v1alpha1/hostedcluster_types.go#L233-L244) you should use